pkg/awsutils/vpc_ip_resource_limit.go

```diff
@@ -19,420 +19,420 @@ package awsutils
 // InstanceNetworkingLimits contains a mapping from instance type to networking limits for the type. Documentation found at
 // https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#AvailableIpPerENI
 var InstanceNetworkingLimits = map[string]InstanceTypeLimits{
-	"a1.2xlarge":    {ENILimit: 4, IPv4Limit: 15, HypervisorType:"nitro"},
```

Comment:
why is this file changed

Reply:
make format, did this. Let me revert this file.